harlow/sampling/fuzzy_lolavoronoi.py

```diff
@@ -147,7 +158,7 @@ def sample(
             )
 
             start_time = time.time()
-            new_points_x = best_new_points(
+            new_points_x = _best_new_points(
```

Comment:
should these changes also be made to the other samplers or only to fuzzy_lolavoronoi?

Reply:
Yes, it is now part of the sampling base_class